apps/frontend/src/pages/[type]/[id].vue

```diff
@@ -1513,6 +1530,9 @@ try {
 		useAsyncData(`project/${projectId.value}/version`, () =>
 			useBaseFetch(`project/${projectId.value}/version`),
 		),
+		useAsyncData(`project/${projectId.value}/version`, () =>
```

Comment:
Needs to be a different key for the useAsyncData, as it will cause a cache conflict

```suggestion
		useAsyncData(`project/${projectId.value}/version/v3`, () =>
```

Reply:
Apparently it doesn't like it when the useAsyncData key is different. It no longer fetches it properly and I couldn't figure why. Do you have any ideas on why thats happening?

Reply:
Ah nvm, I see some funky stuff going on with versions being turned into shallowRef. Should be working now